lib/apirouter.php

```diff
@@ -84,4 +85,20 @@ private function router() {
 
 		$response->response->send();
 	}
-}
\ No newline at end of file
+    
+    /**
+     * Метод, который забирает роуты и модули, указанные в соотв. классе метода рейтинга webpractik.rating https://github.com/webpractik/webpractik.rating
+     */
+    private function expansionOptions()
+    {
+        $event = new \Bitrix\Main\Event('webpractik.rating', 'OnWebpractikRouterCall');
```

Comment:
Здесь должен быть webpractik.api. Webpractik.api вообще ничего не должен знать о модуле рейтинга.
```suggestion
        $event = new \Bitrix\Main\Event('webpractik.rating', 'OnWebpractikRouterCall');
```